src/main/kotlin/com/github/hanseter/json/editor/SchemaNormalizer.kt

```diff
@@ -245,7 +245,7 @@ object SchemaNormalizer {
             merge(copyTarget().getJSONObject("properties"), propObj.getJSONObject("properties"))
             propObj.optJSONArray("order")?.also { order.put(it) }
             propObj.optJSONObject("order")?.also { order.put(it) }
-            merge(copyTarget(), propObj, KEYS_NOT_TO_MERGE)
+            merge(copyTarget(), propObj, propObj.keySet())
```

Comment:
This would mean nothing is going to be merged. To not override the values already present in the copy target I would propose this:
```suggestion
            merge(copyTarget(), propObj, copyTarget().keySet())
```

Also the constant KEYS_NOT_TO_MERGE can be removed I guess.

Reply:
Oops. Fixed. I blame the "wrong" parameter order on the `merge` function :grinning:

Reply:
That's okay.